stdlib/LinearAlgebra/src/schur.jl

```diff
@@ -143,7 +143,12 @@ schur(A::StridedMatrix{<:BlasFloat}) = schur!(copy(A))
 schur(A::StridedMatrix{T}) where T = schur!(copy_oftype(A, eigtype(T)))
 
 schur(A::Symmetric) = schur(copyto!(similar(parent(A)), A))
-schur(A::Hermitian) = schur(copyto!(similar(parent(A)), A))
+schur(A::Symmetric{<:Real}) = _schur(A)
+schur(A::Hermitian) = _schur(A)
+function _schur(A::RealHermSymComplexHerm)
```

Comment:
Can't this just be `function schur(A::RealHermSymComplexHerm)`?   Why do you need a separate `_schur` function here?

Reply:
Because otherwise I was getting an ambiguity error with `schur(A::Symmetric)`.